### PR TITLE
Fix links in Chinese translation

### PR DESCRIPTION
**Pull request by tengqm**
As a follow up of #6454

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/website/6467)

### CODE REVIEW

File: cn/docs/concepts/overview/what-is-kubernetes.md [hunk 1]
```diff
@@ -70,7 +70,7 @@ Kubernetes 满足了生产中运行应用程序的许多常见的需求，例如
 * [Pod](/docs/user-guide/pods/) 提供复合应用并保留一个应用一个容器的容器模型,
 * [挂载外部存储](/docs/user-guide/volumes/),
```
Comment (xiangpengzhao) [inline cn/docs/concepts/overview/what-is-kubernetes.md]: I get a 404 here.

File: cn/docs/concepts/overview/what-is-kubernetes.md [hunk 2]
```diff
@@ -70,7 +70,7 @@ Kubernetes 满足了生产中运行应用程序的许多常见的需求，例如
...
 * [Secret管理](/docs/user-guide/secrets/),
-* [应用健康检查](/docs/user-guide/production-pods/#liveness-and-readiness-probes-aka-health-checks),
+* [应用健康检查](/docs/tasks/configure-pod-container/configure-liveness-readiness-probes/),
```
Comment (xiangpengzhao) [inline cn/docs/concepts/overview/what-is-kubernetes.md]: A 404 as well.

File: cn/docs/concepts/overview/what-is-kubernetes.md [hunk 3]
```diff
@@ -70,7 +70,7 @@ Kubernetes 满足了生产中运行应用程序的许多常见的需求，例如
...
 * [副本应用实例](/docs/user-guide/replication-controller/),
 * [横向自动扩缩容](/docs/user-guide/horizontal-pod-autoscaling/),
 * [服务发现](/docs/user-guide/connecting-applications/),
```
Comment (xiangpengzhao) [inline cn/docs/concepts/overview/what-is-kubernetes.md]: 404. Please also check other links around here.

File: cn/docs/concepts/overview/what-is-kubernetes.md (hunk 1 above)
Comment (tengqm) [reply cn/docs/concepts/overview/what-is-kubernetes.md]: fixed

File: cn/docs/concepts/overview/what-is-kubernetes.md (hunk 2 above)
Comment (tengqm) [reply cn/docs/concepts/overview/what-is-kubernetes.md]: this should work

File: cn/docs/concepts/overview/what-is-kubernetes.md (hunk 3 above)
Comment (tengqm) [reply cn/docs/concepts/overview/what-is-kubernetes.md]: these links are taken care of by the redirection

File: cn/docs/concepts/overview/what-is-kubernetes.md (hunk 1 above)
Comment (tengqm) [reply cn/docs/concepts/overview/what-is-kubernetes.md]: actually this one is a redirection as well. so I'm not submitting another commit.

Comment (xiangpengzhao) [reply cn/docs/concepts/overview/what-is-kubernetes.md]: cool. where is this link redirected to ?

Comment (tengqm) [reply cn/docs/concepts/overview/what-is-kubernetes.md]: https://github.com/kubernetes/website/blob/master/_redirects#L390

**Comment by tengqm** [comment]
@xiangpengzhao can you verify that the reported 404 is still a problem? The links work for me.

**Comment by xiangpengzhao** [comment]
@tengqm cool! The aforementioned links work well on https://kubernetes.io/docs.

**Pull request merged**